dev/ScrollBar/ScrollBar_themeresources.xaml

```diff
@@ -100,8 +100,8 @@
             <StaticResource x:Key="ScrollBarTrackStroke" ResourceKey="SystemControlForegroundTransparentBrush" />
             <StaticResource x:Key="ScrollBarTrackStrokePointerOver" ResourceKey="SystemControlForegroundTransparentBrush" />
             <StaticResource x:Key="ScrollBarTrackStrokeDisabled" ResourceKey="SystemControlDisabledTransparentBrush" />
-            <SolidColorBrush x:Key="ScrollBarThumbBackground" Color="{StaticResource ScrollBarThumbBackgroundColor}" />
-            <SolidColorBrush x:Key="ScrollBarPanningThumbBackground" Color="{StaticResource ScrollBarPanningThumbBackgroundColor}" />
+            <StaticResource x:Key="ScrollBarThumbBackground" ResourceKey="SystemControlForegroundChromeDisabledLowBrush" />
```

Comment:
>StaticResource [](start = 13, length = 14)

@mikehillberg is it okay to change the type of these resources?

Reply:
@ranjeshj too

Reply:
I think this should be ok since this is not source breaking, but @MikeHillberg can confirm if this is ok.